crates/chia-consensus/src/spendbundle_conditions.rs

```diff
@@ -262,6 +272,92 @@ mod tests {
         SpendBundle::new(spends, Signature::default())
     }
 
+    // this function compares the expected and actual outputs
```

Comment:
I think it would be good to explain a bit more about what this function does. Since it's only called from one place, I imagine it's mostly here to help organize and explain the code. However, it takes a lot of input, so it's not so obvious what it does. `a1` and `a2` could use an explanation for instance.